release-notes/10.0/preview/preview1/README.md

```diff
@@ -0,0 +1,37 @@
+# .NET 10 Preview 1 - Release Notes
+
+.NET 10 Preview 1 released on February X, 2025. Find more information on new features released in .NET 10 Preview 1 by browsing through the release notes below:
+
+- [Libraries](./libraries.md)
+- [Runtime](./runtime.md)
+- [SDK](./sdk.md)
+
+## Feature Release Notes
+
+- [.NET Aspire](./dotnetaspire.md)
+- [.NET MAUI](./dotnetmaui.md)
+- [ASP.NET Core](./aspnetcore.md)
+- [Container images](./containers.md)
+- [EF Core & Data](./efcore.md)
+- [Windows Forms](./winforms.md)
+- [WPF](./wpf.md)
+
+## Get Started
+
+Instructions on getting started with .NET 10 can be found in the [getting started guide](../../get-started.md). Installers and binaries for .NET 10 Preview 1 can be found [here on GitHub](./10.0.0-preview.1.md).
+
+## Stay up-to-date
+
+You can find a detailed overview of all new features in .NET 10:
+
+- [What's new in .NET 10](https://learn.microsoft.com/dotnet/core/whats-new/dotnet-10/overview)
+- [What's new in C# 14](https://learn.microsoft.com/dotnet/csharp/whats-new/csharp-14)
+- [What's new in F# 10](https://learn.microsoft.com/dotnet/fsharp/whats-new/fsharp-10)
+- [What's new in .NET Aspire](https://learn.microsoft.com/dotnet/aspire/whats-new/dotnet-aspire-10)

```

Comment:
```suggestion
```
Were we going to remove .NET Aspire from this release announcement since it's a separate release? If not, needs to point to .NET Aspire 9.1 instead of .NET Aspire 10 for the link.